packages/preview/curryst/0.4.0/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2025 Rémi Hutin - Paul Adam
```

Comment:
You maybe want to add your name here @MDLC01 if you contributed a lot of code to this release.